spec/models/course/requirement/achievement_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'rails_helper'
 
 RSpec.describe Course::Condition::Achievement, type: :model do
-  it { is_expected.to act_as(:condition).class_name(Course::Condition.name) }
+  it { is_expected.to acts_as(:condition).class_name(Course::Condition.name) }
```

Comment:
No problem, I also didn't understand the rationale until @jsyeo explained here >.<